README.md

```
@@ -488,12 +502,14 @@ npm run verify:emilia-map -- --data-root C:\Users\dacan\GeoLens\data\emilia-roma
 
 ## External evidence intake
 
-GeoLens includes one local intake command for future ARPAE and Amsterdam/Waternet deliveries. It reads originals from a data directory outside Git, computes their byte counts and SHA-256 identities, validates the appropriate scientific contract and writes a new package receipt. It does not copy the originals and refuses to overwrite an existing receipt.
+GeoLens includes one local intake command for future ARPAE, Amsterdam/Waternet and Cumbria model deliveries. It reads originals from a data directory outside Git, computes their byte counts and SHA-256 identities, validates the appropriate scientific contract and writes a new package receipt. It does not copy the originals and refuses to overwrite an existing receipt.
 
 ```powershell
```

markdownlint-cli2 (0.23.2)

[warning] Code fence style
Expected: tilde; Actual: backtick

(MD048, code-fence-style)